### PR TITLE
Fix showing user index id and win emojis

### PR DESCRIPTION
**Pull request by huybt22**
~~- Fix wrong players index input~~
~~- Now everything related to index of player should use on `all players list`~~
~~- For action curse, the dead list also includes player ID~~
~~- Remove all `!` in mentions since it's deprecated~~

- Fix to show correct user index id in all players list, move dead players to bottom
- Fix to also show win emoji for winning couple
- Now new day time phase should use all players list
- Refactor game mode commands

image

image

### CODE REVIEW

**Comment by huybt22** [comment]
Any code block contains `get_alive_players()` should be checked carefully.

File: commands/command.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ async def parse_command(client, game, message):
                     is_valid_command = False
                     if all(param.isdigit() for param in parameters):
                         targets_index = [int(param) - 1 for param in parameters]
-                        id_players = game.get_dead_players() if cmd == "reborn" else game.get_alive_players()
+                        id_players = game.get_all_players()
```
Comment (nitro2) [inline commands/command.py:103]: What happens if reborn an alive player?

**Review by nitro2 — CHANGES_REQUESTED** [review]
It's better to create player_index in Player class then reuse it. Do not replace all `get_alive_players()` and `get_dead_players()` by `get_all_players()` . It can introduce newer bugs.

File: game/__init__.py [hunk 2]
```diff
@@ -1021,18 +1021,21 @@ def is_alive():
...
 
         @check(is_alive)
         @check(is_night)
         @check(has_role(roles.Seer))
         async def auto_seer():
-            target = random.choice(self.get_alive_players())
-            if author.player_id in self.cupid_dict:
-                while target.player_id in  self.cupid_dict:
-                    target = random.choice(self.get_alive_players())
+            alives = self.get_all_players()
```
Comment (nitro2) [inline game/__init__.py:1035]: Should seer only alive players

File: game/__init__.py [hunk 3]
```diff
@@ -1021,18 +1021,21 @@ def is_alive():
         @check(is_night)
         @check(has_role(roles.Guard))
         async def auto_guard():
-            target = random.choice(self.get_alive_players())
+            alives = self.get_all_players()
```
Comment (nitro2) [inline game/__init__.py:1024]: Should gaurd only alive players

**Comment by huybt22** [comment]
This PR has been fixed base on the new `main` and passed checking jobs, please have a look. @nitro2

**Review by neihousaigaai — APPROVED** [review]

File: game/text_template.py [hunk 4]
```diff
@@ -179,33 +192,31 @@ def generate_help_embed(*args):
 
 
 def generate_on_off_value(str_value):
-    if str_value == 'True':
+    if str_value in ['True', 'on']:
         return text_templates.get_word_in_language("turn_on").capitalize()
 
-    if str_value == 'False':
+    if str_value in ['False', 'off']:
```
Comment (nitro2) [inline game/text_template.py:198]: nice method :D

**Review by nitro2 — APPROVED** [review]

**Pull request merged**